app/src/sharedTest/java/org/oppia/app/parser/HtmlParserTest.kt

```diff
@@ -47,7 +47,7 @@ class HtmlParserTest {
   @Test
   fun testHtmlContent_handleCustomOppiaTags_parsedHtmlDisplaysStyledText() {
     val textView = activityTestRule.activity.findViewById(R.id.test_html_content_text_view) as TextView
-    val htmlParser = htmlParserFactory.create(/* entityType= */ "", /* entityId= */ "")
+    val htmlParser = htmlParserFactory.create(/* entityType= */ "", /* entityId= */ "", /* imageCenterAlign= */ false)
```

Comment:
Here it should be true. As it is content and not item selection.

Reply:
Done.